src/gui/transferlistwidget.cpp

```diff
@@ -550,14 +554,22 @@ void TransferListWidget::openSelectedTorrentsFolder() const
     QSet<QString> pathsList;
     foreach (BitTorrent::TorrentHandle *const torrent, getSelectedTorrents()) {
         QString path = torrent->contentPath(true);
+#ifndef Q_OS_MAC
```

Comment:
Instead of injecting `#if` in the middle, I suggest separate the code path:
```c++
#ifdef Q_OS_MAC
QSet<QString> pathsList;
foreach(){}
openFiles(pathsList);
#else
QSet<QString> pathsList;
foreach(){}
#endif
```

Reply:
Don't mind with either, will do.